device/arista/x86_64-arista_7800r3_48cq2_lc/Arista-7800R3-48CQ2-C48/sai_postinit_cmd.soc

```diff
@@ -0,0 +1,28 @@
+INTeRrupt ENAble id=1550
+INTeRrupt ENAble id=1551
+INTeRrupt ENAble id=1552
+INTeRrupt ENAble id=1553
+INTeRrupt ENAble id=1554
+INTeRrupt ENAble id=1555
+INTeRrupt ENAble id=1556
+INTeRrupt ENAble id=1557
+INTeRrupt ENAble id=1558
+INTeRrupt ENAble id=1559
+INTeRrupt ENAble id=1560
+INTeRrupt ENAble id=1561
+INTeRrupt ENAble id=1562
+INTeRrupt ENAble id=1563
+INTeRrupt ENAble id=1564
+INTeRrupt ENAble id=1565
+INTeRrupt ENAble id=1566
+INTeRrupt ENAble id=1567
+
+INTeRrupt ENAble id=1296
+INTeRrupt ENAble id=1297
+INTeRrupt ENAble id=1298
+INTeRrupt ENAble id=1299
+INTeRrupt ENAble id=1661
```

Comment:
Is it necessary to monitor 1bit errors? If the underlying memories have ECC, 1 bit errors should be benign?

Reply:
@kenneth-arista, these interrupts are for cases which cause packet drops because of packet corruption. Sent you the details offline